src/thing_list.c

```diff
@@ -3566,6 +3566,7 @@ struct Thing *get_nearest_object_at_position(MapSubtlCoord stl_x, MapSubtlCoord
         }
       }
     }
+    mapblk += 320;
```

Comment:
Again a weird number, what does this +320 do here?

Reply:
I've no idea; that's just what's in the original code.